packages/common/src/store/pages/chat/slice.ts

```diff
@@ -284,8 +296,22 @@ const slice = createSlice({
     ) => {
       // triggers saga to get chat if not exists
       const { chatId, message, status } = action.payload
+
+      // Recalculate hasTail of previous message
+      const prevLatestMessageId = state.messages[chatId].ids[0]
```

Comment:
assuming ids will always have length > 0 here?

Reply:
So I think it'll just come out to `undefined` if not, which is fine because then the following if statement will fail.